src/main/java/datawave/data/type/ListType.java

```diff
@@ -44,4 +44,9 @@ public void setDelegateFromString(String in) {
     public List<String> getNormalizedValues() {
         return normalizedValues;
     }
+    
+    @Override
+    public boolean expandAtQueryTime() {
+        return false;
```

Comment:
Should the default for a list type not be true?

Reply:
I debated that...  My thought at the time was to wait until we had a use case that proved we needed it for all list types, but right now the only one we know we need it for is the NumberListType.  I could go either way.